gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+
+android.useDeprecatedNdk=true
```

Comment:
With the upgraded android build tools, you have to add the "android.useDeprecatedNdk=true" flag in order to continue using the NDK tools with newer versions of build tools. Ultimately, they've deprecated this and are trying out a new experimental build system that I don't believe we are quite ready to switch over to yet.